baseapp/options.go

```diff
@@ -21,6 +21,29 @@ func SetPruning(opts pruningtypes.PruningOptions) StoreOption {
 	return func(config *multi.StoreParams, _ uint64) error { config.Pruning = opts; return nil }
 }
 
+func SetSubstoreTracer(w io.Writer) StoreOption {
```

Comment:
These exported functions should be commented:
- `SetSubstoreTracer`
- `SetTracerFor`

Reply:
Documented

Reply:
Documented